labs/lab9/static/css/style.css

```diff
@@ -0,0 +1,3 @@
+;nothing here
```

Comment:
I feel it would be easy to write something to use these 2 static files that are otherwise empty

Reply:
Yeah, I had plans to. But with limited time, I'd figure I work on the main stuff first.